.github/workflows/publish_pages.yml

```diff
@@ -0,0 +1,46 @@
+name: Publish Pages
+
+on: 
+  push:
+    branches:
+      - master
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      
+    - uses: actions/setup-node@v2
+      with:
+        node-version: 10.x
+    
+    - uses: actions/setup-python@v2
+      with:
+        python-version: '3.6.x' 
```

Comment:
Can we add `.node-version` and `.python-version` files (or equivalents) instead of hardcoding the versions in the actions?

Reply:
@lfdebrux I have addressed your comments now, please may you re-review? Note: for python I thought it simpler to read the file in rather than using a 3rd party action but let me know if you feel strongly against this.

Reply:
Works for me